src/modules/SdnDiag.Common/public/Start-SdnDataCollection.ps1

```diff
@@ -102,19 +102,22 @@ function Start-SdnDataCollection {
         [bool]$ConvertETW = $true
     )
 
+    $dataCollectionNodes = New-Object System.Collections.ArrayList
+    $filteredDataCollectionNodes = @()
```

Comment:
May want to make style consistent to init this as ArrayList as well?

Reply:
So in this, we will have a mix of arrays and arraylists. I am using an arraylist on $dataCollectionNodes only for the purpose of being able to remove objects from the arraylist, which cannot be done w/ a standard array. I can add a comment to explain the need for this.